Tasks/Common/kubernetes-common-v2/module.json

```diff
@@ -3,6 +3,7 @@
         "UnableToFetchKubectlVersion": "Could not fetch Kubectl version. Please make sure that the Kubernetes server is up and running.",
         "KubectlClientVersion": "Kubectl Client Version",
         "KubectlServerVersion": "Kubectl Server Version",
-        "KubectlServerVerisonNotFound": "Could not find kubectl server version"
+        "KubectlServerVerisonNotFound": "Could not find kubectl server version",
+        "HelmLatestNotKnown": "Cannot get the latest Helm info from %s. Error %s. Using default Helm version %s."
```

Comment:
This is not required

Reply:
In case of errors, the task throws a warning that 'HelmLatestNotKnown' not found in loc (something like that). It's required for tl.loc(), not for telemetry.

Reply:
cool